readme.md

```diff
@@ -127,6 +127,20 @@ Custom promise module to use instead of the native one.
 - Pify has useful options like the ability to handle multiple arguments (`multiArgs`).
 - Pify does not have [magic behavior](https://nodejs.org/api/util.html#util_custom_promisified_functions) for certain Node.js methods and instead focuses on predictability.
 
+#### Why can't a promisified class function access it's own context?
+
+The context of a function can't easily be detected. As a workaround, promisify the whole class and extract the function from the result.
+
+```js
+const pify = require('pify');
+const SomeClass = require('./some-class');
+
+const someClass = new SomeClass();
+
+// `someFunction` is extracted from the entire promisified class, allowing it to keep it's context
+const {someFunction} = pify(someClass);
```

Comment:
This will not work, for the same reason that `const { someFunction } = someClass` will not work. JavaScript functions are not bound, it is the method call syntax (`obj.fun(args)`) that sets the value of `this`, so the promisified function has to be called via a method call either on the original object or the one wrapped with `pify`.

Reply:
Fixed!